views/extras.tmpl

```diff
@@ -353,3 +353,10 @@
   data-rev-total="{{$likely.RevokeTotal}}"
   data-rev-count="{{.NumRevokes}}"
 {{end}}
+
+{{define "copyTextIcon"}}
+	&nbsp;
+	<span class="dcricon-copy clickable"
+	data-controller="extras"
```

Comment:
I know it's a WIP and you may not be this far yet, but I just wanted to make sure you didn't write this controller but forget to `git add` it.

Reply:
I wrote it it just wasn't presentable yet just a bunch of console logs and window alerts